src/tools/rbd/action/DiskUsage.cc

```diff
@@ -77,8 +81,13 @@ static int compute_image_disk_usage(const std::string& name,
     if (!snap_name.empty()) {
       full_name += "@" + snap_name;
     }
-    tbl << full_name
-        << stringify(byte_u_t(size))
+    tbl << full_name;
+    if (snap_name.empty())
+      tbl << id;
+    else
+      tbl << snap_id;
```

Comment:
Why would this display the snapshot id? Seems odd to switch from image id to snap id for the same column.